tree/dataframe/test/dataframe_cache.cxx

```diff
@@ -251,6 +251,51 @@ TEST(Cache, Regex)
    auto cached = tdfs.Cache();
    auto m = cached.Max<ULong64_t>("col0");
    EXPECT_EQ(3UL, *m);
+
+   // second part
```

Comment:
leftover comment?

Reply:
not really, but if you want this can be removed :)

Reply:
it's a small thing, but i don't understand the comment then